src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-your-own.mdx

```diff
@@ -1,23 +1,17 @@
 ---
-title: Instrument your own functions
-tags:
-  - Serverless function monitoring
-  - AWS Lambda monitoring
-  - Enable Lambda monitoring
-  - Instrumentation
+title: "Enable Lambda monitoring step 3: Instrument your own functions"
```

Comment:
@zuluecho9, could we shorten this title to this: "Step 3: Review Lambda function instrumentation examples"? I'm guessing that users can get the context from the title and from the left navigation.